src/Mpdf.php

```diff
@@ -12970,12 +12977,18 @@ function SetFooter($Farray = [], $side = '')
 
 	/* -- WATERMARK -- */
 
-	function SetWatermarkText($txt = '', $alpha = -1)
+	function SetWatermarkText($txt = '', $alpha = -1, $size = 120, $pos = null)
```

Comment:
A nice attempt, but the same applies in this case too: this would be technically a BC break and applicable the soonest in 9.0.